### PR TITLE
QGB depend on latest version of core and use begin/end block instead of query for commitments

### PR DESCRIPTION
**Pull request by SweeXordious**
Closes https://github.com/celestiaorg/celestia-app/issues/604

### CODE REVIEW

File: x/qgb/orchestrator/orchestrator.go [hunk 1]
```diff
@@ -345,7 +345,8 @@ func (orch Orchestrator) ProcessDataCommitmentEvent(
 ) error {
 	commitment, err := orch.Querier.QueryCommitment(
 		ctx,
-		CommitmentQueryByRange(dc.BeginBlock, dc.EndBlock),
+		dc.BeginBlock,
+		dc.EndBlock,
```
Comment (SweeXordious) [inline x/qgb/orchestrator/orchestrator.go:349]:
```suggestion
		dc.EndBlock,
		
```

**Review by SweeXordious — COMMENTED** [review]
adding dummy change for CI to be triggered

File: x/qgb/orchestrator/orchestrator.go [hunk 2]
```diff
@@ -345,7 +345,9 @@ func (orch Orchestrator) ProcessDataCommitmentEvent(
...
+
```
Comment (SweeXordious) [inline x/qgb/orchestrator/orchestrator.go:350]:
```suggestion
```

File: x/qgb/orchestrator/querier.go [hunk 3]
```diff
@@ -259,11 +265,14 @@ func (q querier) QueryTwoThirdsValsetConfirms(
 			}
 			q.logger.Debug(
 				"foundValsetConfirms",
```
Comment (SweeXordious) [inline x/qgb/orchestrator/querier.go:267]:
```suggestion
				"found ValsetConfirms",
```

File: x/qgb/orchestrator/querier.go [hunk 4]
```diff
@@ -179,11 +179,17 @@ func (q *querier) QueryTwoThirdsDataCommitmentConfirms(
...
 			q.logger.Debug(
-				fmt.Sprintf(
-					"found DataCommitmentConfirms total power %d number of confirms %d",
-					currThreshHold,
-					len(confirms),
-				),
+				"found DataCommitmentConfirms",
+				"begin_block",
+				dc.BeginBlock,
+				"end_bloc",
```
Comment (rootulp) [inline x/qgb/orchestrator/querier.go:185]:
```suggestion
				"end_block",
```

File: x/qgb/orchestrator/querier.go [hunk 5]
```diff
@@ -258,12 +264,15 @@ func (q querier) QueryTwoThirdsValsetConfirms(
...
-				fmt.Sprintf(
-					"total power %d number of confirms %d",
-					currThreshHold,
-					len(confirmsResp.Confirms),
-				),
+				"found ValsetConfirms",
+				"nonce",
+				valset.Nonce,
+				"total_power",
+				currThreshHold,
```
Comment (rootulp) [inline x/qgb/orchestrator/querier.go:271]: typo: currThreshHold -> currThreshold

File: x/qgb/orchestrator/test/test_utils.go [hunk 6]
```diff
@@ -90,6 +90,6 @@ func verifyOrchestratorDcSignature(broadcasted sdk.Msg, dc types.DataCommitment)
 	return nil
 }
 
-func commitmentFromQuery(query string) []byte {
-	return tmhash.Sum([]byte(query))
+func commitmentFromQuery(beginBlock uint64, endBlock uint64) []byte {
```
Comment (rootulp) [inline x/qgb/orchestrator/test/test_utils.go:93]: [question] is this function name still appropriate given the parameter change?

**Review by rootulp — COMMENTED** [review]
No blocking feedback but I'm not sure what in this PR fixes https://github.com/celestiaorg/celestia-app/issues/604 so defer to @evan-forbes

**Review by evan-forbes — APPROVED** [review]
:+1:

**Comment by evan-forbes** [comment]
> No blocking feedback but I'm not sure what in this PR fixes https://github.com/celestiaorg/celestia-app/issues/604

for posterity: the fix is in core https://github.com/celestiaorg/celestia-core/pull/838

File: x/qgb/orchestrator/test/test_utils.go (hunk 6 above)
Comment (SweeXordious) [reply x/qgb/orchestrator/test/test_utils.go:93]: thanks, changed.

**Pull request merged**